mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -584,12 +591,11 @@ func resourceStorageBucketUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
-	if d.HasChange("uniform_bucket_level_access") {
+	if d.HasChange("uniform_bucket_level_access") <% unless version == "ga" -%> || d.HasChange("public_access_prevention") <% end -%> {
```

Comment:
No need to gate this check based on the provider? I guess if we make the other change you suggested, so the schema for the two align, are we then assuming that no change will ever be set for the GA provider, making this check redundant?

Reply:
I'm resolving this as I think this was down to the confusion over whether this is a beta or a GA feature (it's the former, title of PR updated to avoid future confusion).